src/util/string.js

```diff
@@ -0,0 +1,2 @@
+export const uintToString = (uintArray) =>
```

Comment:
Minor, but this function is doing a bit more than just converting the typed array to a string. We may want to either rename the function or just call decodeURIComponent and escape within vtt-segment-loader.